imaginairy/api.py

```diff
@@ -204,7 +204,7 @@ def imagine(
                     generation_strength = 1 - prompt.init_image_strength
                     ddim_steps = int(prompt.steps / generation_strength)
                     sampler.make_schedule(ddim_num_steps=ddim_steps, ddim_eta=ddim_eta)
-                    init_image, _, h = pillow_fit_image_within(
+                    init_image, _ = pillow_fit_image_within(
```

Comment:
this will def break things... the function returns 3 values. where does the 3rd go now?